### PR TITLE
reloading workflow

### PR DESCRIPTION
**Pull request by njackson**

### CODE REVIEW

**Comment by mwynholds** [comment]
Is there a reason why we wouldn't just add this Procfile.dev to the repo (and take it out .gitignore), and add nodemon to the devDependencies? @njackson

**Comment by njackson** [comment]
Some folks might want something different in Procfile.dev, for example Sidney said he runs mongo and redis servers through launchd, whereas I don't like having them running and would prefer to launch them manually. If everybody wants this as their Procfile then I'm all for committing it.

**Comment by mwynholds** [comment]
Got it.  That makes sense.  I run those things via launchd as well, so I guess this is the best way.

LGTM

**Comment by njackson** [comment]
Talked to Sidney, he said he's down for not doing the launchd thing and putting this in. You can keep that if you want, but you'd have to make your own Procfile to not launch db servers that already are running.

File: README.md [hunk 1]
```diff
@@ -38,6 +38,11 @@ testing) use "--coffee" to enable CoffeeScript support
 - Cakefile: "cake spec:client", "cake spec:server", "cake spec"
 - npm: "npm test" is also available for running all tests
 
+
+### Run everything with automatic reloading
+
+`foreman start -f Procfile.dev`
```
Comment (andrewhao) [inline README.md]: Do you mean `nf` instead of `foreman`?

**Comment by sidke** [comment]
:+1:

**Pull request merged**